spec/spec_helper.rb

```diff
@@ -14,6 +14,8 @@
 require "topological_inventory/core/ar_helper"
 TopologicalInventory::Core::ArHelper.load_environment!
 
+Rails.logger = Logger.new(TopologicalInventory::Core::ArHelper.root.join("log/test.log"))
```

Comment:
Is this necessary?

Reply:
Yes, that's the entire purpose of the PR ... The specs were failing before with no method `#info` for nil.

Reply:
Oh, I didn't know that

Reply:
Ah, sorry, `s/PR/commit/` 

Originally I just added the logging to the method, but then the specs started failing so I had to do this.